cypress/support/fragments/orders/orders.js

```diff
@@ -131,5 +131,28 @@ export default {
 
   closeThirdPane: () => {
     cy.do(PaneHeader({ id: 'paneHeaderorder-details' }).find(Button({ icon: 'times' })).click());
-  }
+  },
+
+  getSearchParamsMap(orderNumber, currentDate) {
+    const searchParamsMap = new Map();
+    searchParamsMap.set('PO number', orderNumber)
+      .set('Keyword', orderNumber)
+      .set('Date created', currentDate);
+    return searchParamsMap;
+  },
+
+  checkPoSearch(searchParamsMap, orderNumber) {
+    for (const [key, value] of searchParamsMap.entries()) {
+      cy.do([
+        SearchField({ id: 'input-record-search' }).selectIndex(key),
```

Comment:
move  SearchField({ id: 'input-record-search' } to constant

Reply:
Updated